tabpy-tools/tabpy_tools/rest.py

```diff
@@ -1,21 +1,25 @@
 import logging as _logging
 from future.utils import with_metaclass
 import requests
+from re import compile as _compile
+
 _logger = _logging.getLogger(__name__)
 
 from collections import MutableMapping as _MutableMapping
 
 import json as json
 
-from tabpy_server.common.tabpy_logging import (
+from .tabpy_logging import (
```

Comment:
Generally I think you should get rid of the custom logging module as it doesn't serve a huge purpose. I have a separate PR for that

However, if you do feel a need to keep it around I think this import is incorrect and will be VERY buggy with some of the other path hackery going on in setup. Almost always preferable in a package of this size to use absolute imports

Reply:
Again you are right and we are going to look at it and maybe remove the logger replacing it in the future. But for now, I am tending to keep it to not make this huge change list even bigger.